pkg/config/app/kuma-dp/config.go

```diff
@@ -363,6 +369,14 @@ func (d *DataplaneRuntime) Validate() error {
 	if d.BinaryPath == "" {
 		errs = multierr.Append(errs, errors.Errorf(".BinaryPath must be non-empty"))
 	}
+	if d.EnvoyXdsTransportProtocolVariant != "" {
+		switch d.EnvoyXdsTransportProtocolVariant {
+		case "DELTA_GRPC":
```

Comment:
I think it's okay to remove `feature-grpc` as @lahabana mentioned, because it's not indeed a "feature". But if the config has the field `ProtocolVariant`, then I believe it should have 2 values `DELTA_GRPC` and `GRPC`. If the value is `DELTA_GRPC` then kuma-dp sends `feature-delta-grpc`, if it's `GRPC` then kuma-dp sends nothing